src/Halogen/HTML/Events.purs

```diff
@@ -48,24 +48,19 @@ module Halogen.HTML.Events
   ) where
 
 import Prelude
-
+import DOM.Event.Event as EE
+import Halogen.HTML.Core as Core
 import Control.Monad.Except (runExcept)
-
-import Data.Either (either)
-import Data.Foreign (toForeign)
-import Data.Foreign.Class (class IsForeign, readProp)
-import Data.Maybe (Maybe(..))
-
 import DOM.Event.Types (ClipboardEvent, Event, EventType(..), FocusEvent, KeyboardEvent, MouseEvent)
-import DOM.Event.Event as EE
 import DOM.HTML.Event.Types (DragEvent)
-
-import Halogen.HTML.Core as Core
+import Data.Either (either)
+import Data.Foreign (Foreign, F, toForeign, readString, readInt, readBoolean)
+import Data.Foreign.Index (readProp)
+import Data.Maybe (Maybe(..))
 import Halogen.HTML.Core (Prop)
 import Halogen.HTML.Properties (IProp)
 import Halogen.Query (Action, action)
 import Halogen.Query.InputF (InputF(..))
-
 import Unsafe.Coerce (unsafeCoerce)
```

Comment:
Again the import sorting - `psc-ide` sorts them in a different way to our preferred style.